OracleManagementAgent/README.md

```diff
@@ -0,0 +1,136 @@
+<!--
+    Copyright (c) 2022, Oracle and/or its affiliates.
+    Licensed under the Universal Permissive License v 1.0 as shown at
+    https://oss.oracle.com/licenses/upl.
+-->
+Oracle Management Agent Container Image
+=====
+This repository contains sample container configurations to facilitate installation and environment setup for DevOps users. This project includes Dockerfiles based on Oracle Linux and Oracle OpenJDK 8.
+
+The certification of Oracle Management Agent in a container does not require the use of any file presented in this
+repository. Customers and users are welcome to use them as starters, and customize/tweak, or create
+from scratch new scripts and Dockerfiles.
+
+
+## How to build and run
+
+Oracle Management Agent images use the Oracle Linux 7 Slim container image as the base image.
```

Comment:
```suggestion
Oracle Management Agent images uses the official `oraclelinux:7-slim` container image as the base image. 
```

Reply:
Though, I'm not entirely sure what the purpose of this sentence is. It doesn't provide any useful information that's not already contained in the `Dockerfile`.

Reply:
It's just an info statement that we found other docker-images projects had in their doc so we kept it thinking it was a convention